homeassistant/components/notify/facebook.py

```diff
@@ -19,6 +20,8 @@
 
 CONF_PAGE_ACCESS_TOKEN = 'page_access_token'
 BASE_URL = 'https://graph.facebook.com/v2.6/me/messages'
+CR_BRD_MSG_URL = 'https://graph.facebook.com/v2.11/me/message_creatives'
+SND_BRD_MSG_URL = 'https://graph.facebook.com/v2.11/me/broadcast_messages'
```

Comment:
Please use a speaking name.

Reply:
Are you asking me to rename the variables and use fully spelled names like: CREATE_BROADCAST_MSG_URL instead of abbreviation ? I can do this, but why is it a problem especially since there are many constants that are abbreviated . even the one above: CONF_PAGE_ACCESS_TOKEN

Reply:
A name should be self explanatory. CONF is easy to understand. CR, SND, BRD isn't.

Reply:
Ok, I have changed named to be more readable. The only reason I was abbreviating was because Travis kept telling me I am running over 80 chars in the line and was NOT allowing me to build without errors.